models/istio_validation.go

```diff
@@ -198,36 +199,44 @@ var checkDescriptors = map[string]IstioCheck{
 		Message:  "KIA0903 ServiceRole does not exists in this namespace",
 		Severity: ErrorSeverity,
 	},
+	"sidecar.selector.workloadnotfound": {
+		Message:  "KIA1001 No matching workload found for authorization policy selector in this namespace",
+		Severity: WarningSeverity,
+	},
+	"sidecar.multimatch": {
+		Message:  "KIA1002 More than one selector-less Sidecar in the same namespace",
+		Severity: ErrorSeverity,
+	},
 	"virtualservices.nohost.hostnotfound": {
-		Message:  "KIA1001 DestinationWeight on route doesn't have a valid service (host not found)",
+		Message:  "KIA1101 DestinationWeight on route doesn't have a valid service (host not found)",
```

Comment:
Let's be careful when changing codes, probably not good idea to update all, I think new codes should be added at the end.

Reply:
I wanted to keep the list of validations sorted alphabetically. I'll keep next new resources at the end, not to force this change.

Reply:
+1 to alphabetizing!

Reply:
These are two different things.
alphabetizing things is good.
But if in version 1.x KIA1001 has a code, you can't change that code in version 1.1 just because we are adding a new element starting by "a".

That's not right.

Otherwise any user would become crazy that KIA1001 in 1.x means something different than KIA1101.

Reply:
@lucasponce Absolutely correct.  Can not change codes.